README.md

```diff
@@ -1,20 +1,37 @@
 
-This template repository equips you with the default initial files required for a Ministry of Justice GitHub repository.
+This template repository equips you with the default initial files required for an Analytical Platform Image Build Template GitHub repository.
+
+This repository is managed in Terraform [here](https://github.com/ministryofjustice/data-platform/blob/main/terraform/github/analytical-platform-repositories.tf#L286).
 
 ## Included Files
 
 The repository comes with the following preset files:
-
+- .GitHub
+  - workflows
+    - built-and-test.yml
+    - release.yml
+    - super-linter.yml
+    - dependency-review.yml
+- scripts
+  - build-and-test.sh
+- test
+  - container-structure-test.yml
+- Dockerfile
+- .EditorConfig
 - LICENSE
 - .gitignore
 - CODEOWNERS
 - dependabot.yml
```

Comment:
- `CODEOWNERS` This goes into `.github`
- dependabot.yml This goes into `.github`